dockstore-cli-integration-testing/generated/src/main/resources/pom.xml

```diff
@@ -46,8 +46,8 @@
       <scope>compile</scope>
       <exclusions>
         <exclusion>
```

Comment:
think the proper order is actually group then artifact, may need to update Maven and/or use mvnw

Reply:
This is the ordering produced from `mvnw`, so perhaps we need to bump the wrapper's version?

Reply:
weird, i used mvnw on develop and it does have these changes

Reply:
Worth a shot. A little puzzled why it was the other order in the first place then, but maybe this repo doesn't have the "fail on changes" check that the main repo does

Reply:
This is odd; we ran into it, I forget which way, when Walt made the dockstore-cli for the hotfix. I don't know what's going on -- what plugin actually does the sorting?

Reply:
Should be Maven itself, looks like 3.8.2 or above